### PR TITLE
Beautify adaptfilter show

### PR DESCRIPTION
**Pull request by GwydionJon**
#18 
Changed axis to have Z facing upwards, removed box and added variable threshold with prior warning.

### CODE REVIEW

**Comment by dokempf** [comment]
The problem is that we do version pinning in `setup.py`: https://github.com/ssciwr/adaptivefiltering/blob/main/setup.py#L13 Removing that constraint should enable installation of version 2 and solve the problem.

**Comment by GwydionJon** [comment]
With the new laspy version the ipyvolume scatter function throws an index error, when directly using self.data.x for the input. Converting this ScaledArrayView object into a numpy array fixes the error and plots the scatter. 
This workaround seems undesirable.

**Comment by dokempf** [comment]
If `laspy` changed their format, we should definitely adapt to it.

**Comment by dokempf** [comment]
For obscure reasons, any changes to `environment.yml` need to be duplicated in `doc/environment-rtd.yml` with a version pin applied. Otherwise, the RTD build runs out of memory in solving the conda environment.

File: adaptivefiltering/dataset.py [hunk 1]
```diff
@@ -14,14 +14,30 @@ def __init__(self, filename):
...
             the current working directory, XDG data directories (Unix only) and the Python package
             installation directory.
+
         :type filename: str
         """
+        # initilize warning threshold to warn the user that show() is not available
+
         filename = locate_file(filename)
-        self.data = laspy.file.File(filename, mode="r")
+        # old laspy style
```
Comment (dokempf) [inline adaptivefiltering/dataset.py:23]: I would prefer outdated code to be removed from the repository. We will not move back to `laspy<2`.

**Review by dokempf — CHANGES_REQUESTED** [review]
Apart from my comment about old code, this is ready to be merged.

**Pull request merged**